account_financial_report_qweb/report/general_ledger_xlsx.py

```diff
@@ -33,22 +33,25 @@ def _get_report_columns(self, report):
             7: {'header': _('Cost center'),
                 'field': 'cost_center',
                 'width': 15},
-            8: {'header': _('Rec.'), 'field': 'matching_number', 'width': 5},
-            9: {'header': _('Debit'),
-                'field': 'debit',
-                'field_initial_balance': 'initial_debit',
-                'field_final_balance': 'final_debit',
-                'type': 'amount',
-                'width': 14},
-            10: {
+            8: {'header': _('Tags'),
+                'field': 'tags',
+                'width': 10},
+            9: {'header': _('Rec.'), 'field': 'matching_number', 'width': 5},
+            10: {'header': _('Debit'),
+                 'field': 'debit',
+                 'field_initial_balance': 'initial_debit',
+                 'field_final_balance': 'final_debit',
+                 'type': 'amount',
+                 'width': 14},
+            11: {
                 'header': _('Credit'),
                 'field': 'credit',
                 'field_initial_balance': 'initial_credit',
                 'field_final_balance': 'final_credit',
                 'type': 'amount',
                 'width': 14
             },
-            11: {'header': _('Cumul. Bal.'),
+            12: {'header': _('Cumul. Bal.'),
```

Comment:
Update indexes for foreign_currency

Reply:
Fixed, and autosquash in the same time.
Thanks for check.